contracts/GuestList.vy

```diff
@@ -157,4 +168,4 @@ def authorized(guest: address, amount: uint256) -> bool:
         return True    
     if block.timestamp > self.activation + self.ape_out:
         return True
-    return self._total_yfi(guest) >= self._time_factor(self.min_bag)
+    return self._total_yfi(guest) >= self._entrance_cost()
```

Comment:
since we are checking a lot of stuff for balance, not sure if its overkill
but if i have only yfi balance or vault i can skip the gas cost payment so something like splitting 
the method `_total_yfi`  into `total_yfi_in_tokens(guest) >= self._entrance_cost()` return right away

else check `total_yfi_in_tokens(guest) + total_yfi_in_liq_pools(guest) >= self_entrance_cost()` 

total_yfi_in_liq_pools(guest) = check maker + uniswap
just thinking that the ones that have YFI in vaults don't need to pay more gas if they dont provide to uniswap or if just checking tokens balance makes the cut.

Maybe the gas cost is not a big deal tho.